content/version/2/0/code_of_conduct.pt-br.md

```diff
@@ -0,0 +1,90 @@
++++
+version = "2.0"
+aliases = ["/version/2/0"]
++++
+
+# Código de Conduta para Colaborantes
+
+## Nossa promessa
+
+Nós, como participantes, contribuintes e lideranças, nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

Comment:
- Contribuinte é quem paga imposto, taxa, etc.

```suggestion
Nós, como participantes, colaboradores e lideranças, nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

Reply:
(Talvez `lideranças` deva ser apenas `liderança`? Ou está no plural de propósito?)

Reply:
Alterei o termo para "colaboradores" aqui também para seguir no padrão.
O plural foi intencional, visto que na versão em inglês está assim ("We as members, contributors, and **leaders**"). Aqui foi apenas o caso de eu querer ser fiel ao texto original. O que você acha?

Reply:
As duas formas estão corretas: "a liderança" refere-se ao grupo de pessoas que exercem a liderança. No contexto de organizações partidárias, é comum falar de "as lideranças". Isso pode se referir a diferentes instâncias, ou também às diferentes pessoas que formam a lidarança.

Reply:
Alterei para o singular em todos os locais.